app/serializers/comment_serializer.rb

```diff
@@ -1,12 +1,18 @@
 class CommentSerializer < ActiveModel::Serializer
   include TagHelper
 
-  attributes :id, :content, :markdown, :commentable_type, :commentable_id, :owner_id, :created_at, :updated_at, :team_id, :owner
+  attributes :id, :content, :markdown, :commentable_type, :commentable_id, :owner_id, :created_at, :updated_at, :team_id, :owner, :attachments
 
   def owner
     ActiveModel::SerializableResource.new(object.owner).serializable_hash[:user]
   end
 
+  def attachments
```

Comment:
`has_many :attachments` wasn't working here, not sure why - this was my workaround to make sure the serializer was used